_common

```diff
@@ -61,7 +61,9 @@ runcurl() {
 comment_on_job() {
     local id=$1 comment=$2 force_result=${3:-''}
     if $enable_force_result && [[ -n $force_result ]]; then
-        comment="label:force_result:$force_result:$comment"
+        comment="label:force_result:$force_result:$comment
+
+$comment"
```

Comment:
I realize this is mentioned in the description... but duplication seems like a bad idea to me. This should be fixed on the openQA side

Reply:
Let's add this change as is for now and discuss a change in openQA later, can we? :)